### DISCUSSION TITLE
Unable to get search page via Lucene

### DISCUSSION THREAD

**Discussion by sergeor** [OrchardCMS/OrchardCore#10600] [Q&A] [answered]
**when I try to access the search page, I get the message «Search is not configured».**

I will describe to you exactly what I do 

- I am creating a new ASP.NET Core empty project. I add reference to `OrchardCore.Application.Cms.Targets`
- I fill the setup page with the recipe `blank site` 
- I'm adding a new Content Type with name `Article`. Add to this, `Title` and `HtmlBody`. Check in Full-text  the `Include display text` and `Include body parts`. Also, in HtmlBody check the `Include this element in the index`, `Stored`, `Analyzed`
- I Create a new Content Type with name `Articles` and I add `title` `HtmlBody` and `List or Article`. I also check all the search related fields as above.
- In the Search section in Lucene Indices, add an index with name Search, standardanalyzer, Any Culture and check Article and Articles 
- In search settings, the index is Search and the Default searched fields is `Content.ContentItem.FullText`. The Allow Lucene queries in search forms is checked

The Run Lucene Query seems to work well. As I get some results

In security Anonymous role I have give access with  check the Query Lucene Search Index and Query Lucene Api

Now when I try to access the search page, I get the message **«Search is not configured».**

After some digging I found in `OrchardCore.Modules\OrchardCore.Lucene\Controllers\SearchController.cs` that the `searchSettings.SearchIndex` is null
Any help?
What I'm doing Wrong

PS: if anyone can provide a video how to configure and use the search with paging with Accent insensitive Case insensitive in non-English language, within Razor pages I will be grateful

**Comment by hishamco** [comment]
Seems the search index doesn't exist, @Skrypt any idea?

**Comment by Skrypt** [comment, marked as answer]
You need to go to the search settings and save the form once. I think the issue is that we don't have defaults settings in that case.

**Comment by sergeor** [comment]
But the form has been saved. When I visit it (search / settings / search) I see that it is there. Default search index = `Search`, Default searched fields = `Content.ContentItem.FullText` and Allow Lucene queries in search forms = `true`

**Reply by Skrypt** [reply]
This is the ViewModel defaults.

**Reply by Skrypt** [reply]
There is probably an issue there but the fact that `searchSettings.SearchIndex = null`  and that Allow Lucene queries in search forms is checked seems to indicate that the form has been saved once but there was no index created beforehand. If this is the case then we should not allow saving this form if there is no index created first.

**Reply by sergeor** [reply]
Ok I will open an issue

**Reply by Skrypt** [reply]
No need, I just tested this and it's fine. It's just a misunderstanding. The search settings can be saved when there is no index created. It saves a null value in the `searchSettings.SearchIndex` and displays a notification message saying this : 

![image](https://user-images.githubusercontent.com/3228637/139596817-bd6aa289-bb68-41e0-81fa-250441ae6a23.png)

Then, the issue is that you created the index after saving the search settings simply.
